src/index.html

```diff
@@ -1,13 +1,353 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBike</title>
     <script src="scripts/main.js" defer></script>
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#">
+              <img
+                class="logo"
+                src="./images/logo.svg"
+                alt="Logo Bike"
+              >
+            </a>
+            <div class="top-actions-right">
+              <a href="tel:+1 234 555-55-55" class="header__call"></a>
+
+              <a href="#menu" class="header__menu-opener"></a>
+            </div>
+          </div>
+          <div class="header__title">
+            Take the Streets
+          </div>
```

Comment:
landing should have h1, maybe here?
```suggestion
          <h1 class="header__title">
            Take the Streets
          </h1>
```